renovate.json5

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Fly by comment, but: In terms of blending this configuration with the docs renovate configuration, I'd suggest looking at [the `federation` repository's approach](https://github.com/apollographql/federation/blob/89193fead2a548d4c8792dcb15564213d9bc817d/renovate.json5).

In particular, where the last element of the `packageRules` array (below) contains a `matchPaths`, [like this](https://github.com/apollographql/federation/blob/89193fead2a548d4c8792dcb15564213d9bc817d/renovate.json5#L100-L106).

To be honest, I'd also suggest changing the `extends` to use (extend) our more global [`apollo-open-source` configuration](https://github.com/apollographql/renovate-config-apollo-open-source/blob/master/package.json):

https://github.com/apollographql/federation/blob/89193fead2a548d4c8792dcb15564213d9bc817d/renovate.json5#L2-L6

Thoughts?

Reply:
done! i think! let me know if you think i missed anything important